app/views/judgment/judgmentUpload.scala.html

```diff
@@ -39,12 +39,13 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
         <form id="file-upload-form" data-consignment-id="@consignmentId">
             <div class="govuk-form-group">
                 <div class="drag-and-drop">
-                    <div class="govuk-summary-list govuk-file-upload">
+                    <div id="success-message-row" class="govuk-summary-list govuk-file-upload">
                         <div class="govuk-summary-list__row">
                             <dd id="folder-selection-success" class="govuk-summary-list__value drag-and-drop__success" hidden
                                 tabindex="-1" role="alert" aria-describedby="success-message-text">
-                                <div>
-                                    <p id="success-message-text">The file "<span id="file-name"></span>" has been selected</p>
+                                <div class="success-message-flexbox-container">
+                                    <p class="success-message-flexbox-item-1" id="success-message-text">The file "<span id="file-name"></span>" has been selected </p>
```

Comment:
There's no css defined for the `success-message-flexbox-item-1` so do we need that class here? I've taken it out and it looks the same as far as I can see. 

If you need it for any reason that's OK but if you don't then you can remove the class from here and also from the places in the tests that it's added as well which should clear things up a bit.